### PR TITLE
refactor: restructure and improve Staking documentation

### PR DESCRIPTION
**Pull request by wischli**
# fixes [KILT ticket #1992](https://github.com/KILTprotocol/ticket/issues/1992)

It won't be easy reviewing this PR as all staking content has been restructured from two files (one for collators, one for delegators) to the following format

1. Become a Collator (*the mandatory setup*)
   1. Overview (*new!* --> https://github.com/KILTprotocol/docs/pull/72/commits/640daaa7f4fbbfdd9de4a96fb3bfc8b155c3ddb3)
   2. Minimum hardware requirements
   3. Session keys
   4. Setup a node
   5. Sync blockchain state
   6. Join collator candidate pool (*screenshots and extrinsic description added*)
2. Advanced Collator Section
   1. Adjust stake
   2. How to exit (*new!*)
   3. Lifecycle of a collator
   4. Monitoring
   5. Bootnodes
   6. Benchmarking
3. Delegate
   1. Overview (*new!* --> https://github.com/KILTprotocol/docs/pull/72/commits/94434777ca7568ec6e55440115d29335a8166b2f)
   2. Become a delegator
   3. Adjust stake
   4. How to revoke
   5. How to exit
   6. Lifecycle of a delegator
4. How to unlock unstaked
5. Troubleshooting (*split into collator and delegator tab except for shared part*)

The majority of the content was completely untouched. Apart from the mentions above, duplicate sections were united. Moreover, a Staking transaction disclaimer, which mentions and links to the Stakeboard, was added to each extrinsic step-by-step guide (--> https://github.com/KILTprotocol/docs/pull/72/commits/4937e1195ae352936abaeb13c15ceda7ef953ea6).

In #87, we added a guide to setup a full node which is very similar to the guide to set up a collator node. However, the latter also relies on session keys and is more complicated. It would not be simple to unite both guides in one builder file. IMO, for now we have to live with that. I did my best to match them up in [4b99410](https://github.com/KILTprotocol/docs/pull/72/commits/4b9941079def21d6ea51c2d7df7ea315677f6366).

### CODE REVIEW

**Comment by weichweich** [comment]
Maybe we can split the guide from being a single page to have multiple sections like the workshop? But I expect that his would require some rewriting work.

**Comment by wischli** [comment]
> Maybe we can split the guide from being a single page to have multiple sections like the workshop? But I expect that his would require some rewriting work.

Yes, that was the intention. The current state of this PR was just a starter to have different sections in separate files.

File: docs/participate/collator/3_setup_node.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+Since the WASM runtime logic is part of the blockchain state itself and hence represents the single source of truth, all collators should execute the WASM version of the runtime logic.
+
+#### Specify the right chain spec
+
+The `--chain` parameter decides which blockchain the KILT collator will join.
+This parameter must be specified for both the parachain and the relaychain, since both chains are, as a matter of fact, separate blockchains.
+The KILT parachain accepts an additional parameter to select the environment to use for the WASM runtime execution.
+This can either be `peregrine` or `spiritnet`.
+
+Hence, to start a collator for the Spiritnet network, the parameter would be `--chain=spiritnet`, while for Peregrine would be `--chain=peregrine`.
```
Comment (ntn-x2) [inline docs/participate/collator/3_setup_node.md]: Right now, `--chain=peregrine` does not work, as there's no hardcoded spec for that.

File: docs/participate/collator/2_session_keys.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Make sure that no unauthorised party is able to access the RPC endpoint of the collator.
+Use SSH forwarding for the RPC port when needing to perform some RPC operations on the node with 
+```
+ssh -L 127.0.0.1:9944:127.0.0.1:9944 <user>@<server>
+```
+
+:::
+
+There are tree ways to create the session keys.
```
Comment (ggera) [inline docs/participate/collator/2_session_keys.md]:
```suggestion
There are three ways to create the session keys.
```

File: docs/participate/collator/3_setup_node.md (hunk 1 above)
Comment (wischli) [reply docs/participate/collator/3_setup_node.md]: Good catch. This was copied over from the existing guide and also lived in the full node tutorial. Fixed in [4b99410](https://github.com/KILTprotocol/docs/pull/72/commits/4b9941079def21d6ea51c2d7df7ea315677f6366).

File: docs/develop/01_sdk/02_Cookbook/04_Issue_and_verify_a_KILT_credential/01_ctypes.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+---
```
Comment (ntn-x2) [inline docs/develop/01_sdk/02_Cookbook/04_Issue_and_verify_a_KILT_credential/01_ctypes.md]: How are these files part of this PR? Was it based on top of some WIP branch?

File: docs/develop/02_chain/04_fullnode.md [hunk 4]
```diff
@@ -59,25 +64,31 @@ values={[
 
 ### Building the full node
 
-Below is the command to build the KILT full node executable.
-The command must be run from the root directory of the repository after it has been cloned.
+In order to build the KILT full node executable, you need to have a [nightly version of Rust](https://www.rust-lang.org/tools/install) and the `wasm32-unknown-unknown` for this toolchain installed.
```
Comment (ntn-x2) [inline docs/develop/02_chain/04_fullnode.md]:
```suggestion
In order to build the KILT full node executable, you need to have a [nightly version of Rust](https://www.rust-lang.org/tools/install) and the `wasm32-unknown-unknown` target for this toolchain installed.
```

File: docs/develop/02_chain/04_fullnode.md [hunk 5]
```diff
@@ -59,25 +64,31 @@ values={[
...
+We recommend to align your nightly version with the one used in the [KILT node repository](https://github.com/KILTprotocol/kilt-node) by executing the [init script](https://github.com/KILTprotocol/kilt-node/blob/develop/scripts/init.sh).
+After cloning the repository, you can build the executable by running the `build` command below from the root directory.
 
 ```bash
+# Clone the repository
+git clone git@github.com:KILTprotocol/kilt-node.git
```
Comment (ntn-x2) [inline docs/develop/02_chain/04_fullnode.md]: C: I would rather use the HTTPS address for general consumption.

File: docs/develop/02_chain/04_fullnode.md [hunk 6]
```diff
@@ -59,25 +64,31 @@ values={[
...
 ./scripts/init.sh
 # Build the executable from source enabling all the optimizations with --release.
 cargo build --release -p kilt-parachain
 ```
 
 :::info
-
-We discourage to use the `develop` branch to build the executable. Instead, the latest commit from `master` should be used.
-
+We discourage to use the default `develop` branch to build the executable.
```
Comment (ntn-x2) [inline docs/develop/02_chain/04_fullnode.md]: S: I know this was taken from the previous content, but we should not simply discourage using `develop`, we should forbid it, as that is indeed a different thing altogether for 99% of the time.

File: docs/develop/02_chain/04_fullnode.md [hunk 7]
```diff
@@ -59,25 +64,31 @@ values={[
...
+Instead, the latest commit from `master` should be used.
```
Comment (ntn-x2) [inline docs/develop/02_chain/04_fullnode.md]: C: I wouldn't instruct them to use the latest commit on master, but rather the commit pointed at by the latest release, perhaps adding a link to it.

File: docs/develop/02_chain/04_fullnode.md [hunk 8]
```diff
@@ -59,25 +64,31 @@ values={[
...
 :::
 
-### An archive node
+The compiled executable can be found in `./target/release/kilt-parachain` after the build process completes successfully.
+
+### Running an archive node
 
-The compiled executable can be found in `./target/release/kilt-parachain` after the build process completes successfully. To run an Archive full node add the option `--pruning archive` to the command.
+To run an Archive full node, add the option `--pruning archive` to the command.
```
Comment (ntn-x2) [inline docs/develop/02_chain/04_fullnode.md:91]: Uppercase or lowercase a for archive? It is used both right now.

File: docs/participate/01_Staking/01_Become a Collator/01_overview.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+title: Overview
+---
+
+## Introduction
+
+Collators are the most important members of the network as they do not only maintain the state by running a full node but also advance it by building state transition proofs and sharing these with the relay chain validators.
+Generally speaking, the latter finalize the proposed block if it is valid.
+It is important to note that elusive collators can never get invalid blocks finalized thanks to the design security umbrella provided by the relay chain.
+Thus, the most harm dishonest collators can do, is to slow down or halt the network.
+As long as there exists at least one honest collator, the parachain is secured and fully operative.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/01_overview.md]: W: Although with a potentially slower block time.

File: docs/participate/01_Staking/01_Become a Collator/01_overview.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+## Introduction
+
+Collators are the most important members of the network as they do not only maintain the state by running a full node but also advance it by building state transition proofs and sharing these with the relay chain validators.
+Generally speaking, the latter finalize the proposed block if it is valid.
+It is important to note that elusive collators can never get invalid blocks finalized thanks to the design security umbrella provided by the relay chain.
+Thus, the most harm dishonest collators can do, is to slow down or halt the network.
+As long as there exists at least one honest collator, the parachain is secured and fully operative. 
+
+Whoever wants to join the KILT network as a collator, has to run a full node of the blockchain and set up their keys.
+You also require a minimum amount of tokens self-staked to qualify for a collator seat.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/01_overview.md]: M: 
```suggestion
You are also required to hold a minimum amount of tokens self-staked to qualify for a collator seat.
```

File: docs/participate/01_Staking/01_Become a Collator/01_overview.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+
+## Introduction
+
+Collators are the most important members of the network as they do not only maintain the state by running a full node but also advance it by building state transition proofs and sharing these with the relay chain validators.
+Generally speaking, the latter finalize the proposed block if it is valid.
+It is important to note that elusive collators can never get invalid blocks finalized thanks to the design security umbrella provided by the relay chain.
+Thus, the most harm dishonest collators can do, is to slow down or halt the network.
+As long as there exists at least one honest collator, the parachain is secured and fully operative. 
+
+Whoever wants to join the KILT network as a collator, has to run a full node of the blockchain and set up their keys.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/01_overview.md]: W: We use you everywhere, I would rephrase this to address the reader.

File: docs/participate/01_Staking/01_Become a Collator/01_overview.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+Then, we go over a few configuration options and show you how to setup and start a KILT collator, including how to generate your sessions keys and join the pool of collator candidates.
+
+:::info
+In case you are already collating, you could check out the advanced section.
+There you will learn how to [**monitor**](../02_Advanced%20Collator%20Section/04_monitoring.md) or [**benchmark**](../02_Advanced%20Collator%20Section/06_benchmarking.md) your node, [**adjust your stake**](../02_Advanced%20Collator%20Section/01_adjust_stake.md), [**fix problems**](../05_troubleshooting.md) or [**leave the network**](../02_Advanced%20Collator%20Section/02_exit.md).
+:::
+
+## Join the Community
+
+As a collator you are obliged to keep track of updates and changes to configuration.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/01_overview.md]: W:
```suggestion
As a collator you are required to keep track of updates and changes to configuration.
```

Sounds a bit less harsh in my opinion 😁

File: docs/participate/01_Staking/01_Become a Collator/01_overview.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+You should also be accessible in case there is any issue with your node as this does not only affect your and your delegator's rewards but also the entire network negatively.
+We recommend to join the [KILT Community Discord server](https://discord.gg/wBrXsB5G) and follow (at least) the **collators** and **staking** channels.
+There, you will receive announcements about future updates and potential mandatory client upgrades.
+Moreover, they serve as a discussion hub for collators and delegators.
+
+After having joined the Discord, feel free to send a DM to `Dudley | KILT protocol#6222` or `William | KILT Protocol#4433` to introduce yourself.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/01_overview.md]: C: Can this be made links that take people directly to your profiles?

File: docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md [hunk 14]
```diff
@@ -0,0 +1,16 @@
+---
+id: hardware-requirements
+title: Minimum hardware requirements
+---
+
+The KILT blockchain extrinsic weights were calculated using the following hardware:
+
+- **OS** - Ubuntu 20.04.2
+- **CPU** - AMD Ryzen 7 1700X
+- **Storage** - A NVMe solid-state drive. Should be reasonably sized to deal with blockchain growth. Starting around 80GB - 160GB will be okay for the first six months of KILT parachain and Kusama relaychain but it will mostly likely grow after that and will have to be re-evaluated on a regular basis.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md]: Haven't we already passed this threshold?

File: docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+- **Memory** - 16GB
+
+Although the aforementioned hardware is by no means the minimum spec required, the new node *should* be at least be as capable as that in all the categories.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md]: It does sound to me a lot like minimum required spec tho 😁

```suggestion
Although the aforementioned hardware is by no means the minimum spec required, the new node *should* be at least as capable as that in all the categories.
```

File: docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+Having more perfoment hardware reduces the probability that the node will not be able to produce and propose a valid block on time during the allocated block production slot, missing out on the collating rewards.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md]:
```suggestion
Having more performant hardware reduces the probability that the node will not be able to produce and propose a valid block on time during the allocated block production slot, missing out on the collating rewards.
```

File: docs/participate/01_Staking/01_Become a Collator/03_session_keys.md [hunk 17]
```diff
@@ -0,0 +1,131 @@
+---
+id: session-keys
+title: Session keys
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Before a collator can author blocks, the node needs to fully sync up with both the parachain and the relaychain.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/03_session_keys.md]: W: 
```suggestion
Before a collator can author blocks, the node needs to fully sync up with both the KILT parachain and the Kusama relaychain.
```

File: docs/participate/01_Staking/01_Become a Collator/03_session_keys.md [hunk 18]
```diff
@@ -0,0 +1,131 @@
...
+Depending on the size of the blockchain states, it may take a number of hours to few days for the node to catch up.
+More details can be found on the [Polkadot network docs](https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-kusama#synchronize-chain-data).
+
+Once the chain has fully synced, the collator needs to set sessions keys to begin collating on the network.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/03_session_keys.md]:
```suggestion
Once the chains has fully synced, the collator needs to set sessions keys to begin collating on the network.
```

I would rather change the phrasing to refer to the collator directly.

File: docs/participate/01_Staking/01_Become a Collator/03_session_keys.md [hunk 19]
```diff
@@ -0,0 +1,131 @@
...
+To check whether the account has already set some session keys, the RPC `hasKey(publicKey, keyType)` and `hasSessionKeys(sessionKeys)` in the `author` can be called.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/03_session_keys.md]: Would it be more clear with

```suggestion
To check whether the account has already set some session keys, the RPC functions `author > hasKey(publicKey, keyType)` and `author > hasSessionKeys(sessionKeys)` can be called.
```

File: docs/participate/01_Staking/01_Become a Collator/03_session_keys.md [hunk 20]
```diff
@@ -0,0 +1,131 @@
...
+The session keys associate a collator node with an account on the blockchain.
+They are hot keys that must be kept online.
+It is recommended to change them throughout sessions.
+:::
+
+## Generate New Session Keys {#generating-session-keys}
+
+:::warning
+
+Make sure that no unauthorised party is able to access the RPC endpoint of the collator.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/03_session_keys.md]: I guess we decided to go with American english, right?

```suggestion
Make sure that no unauthorized party is able to access the RPC endpoint of the collator.
```

File: docs/participate/01_Staking/01_Become a Collator/04_setup_node.md [hunk 21]
```diff
@@ -0,0 +1,168 @@
+---
+id: setup-node
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/04_setup_node.md:2]: Q: Should this session rather go before setting/rotating session keys?

File: docs/participate/01_Staking/01_Become a Collator/04_setup_node.md [hunk 22]
```diff
@@ -0,0 +1,168 @@
...
+
+```bash
+# Clone the repository
+git clone git@github.com:KILTprotocol/kilt-node.git
+# Check out master branch
+git checkout master
+# Set up the build environment by installing the Rust compiler.
+./scripts/init.sh
+# Build the executable from source enabling all the optimizations with --release.
+cargo build --release -p kilt-parachain
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/04_setup_node.md:117]: I feel the same comments apply here than apply in the fullnode section.

File: docs/participate/01_Staking/01_Become a Collator/05_sync_blockchain.md [hunk 23]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/05_sync_blockchain.md]: S: I would at least merge this section with the node setup part. I can't think of a strong reason to have this small section separate.

File: docs/participate/01_Staking/01_Become a Collator/03_session_keys.md [hunk 24]
```diff
@@ -0,0 +1,131 @@
+---
+id: session-keys
+title: Session keys
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/03_session_keys.md]: C: We typically use an action as part of the title. So here we could do something like `Set and rotate session keys`.

Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/03_session_keys.md]: C: We typically use an action as part of the title. So here we could do something like `Set and rotate session keys`.

File: docs/participate/01_Staking/01_Become a Collator/06_join_collators.md [hunk 25]
```diff
@@ -0,0 +1,58 @@
...
+
+import StakingTxDisclaimer from '../_disclaimer_staking_tx.md';
+
+After you have finished with the setup, you can finally tell the chain that you are ready to collate and join the pool of candidates.
+
+:::warning
+These steps should be followed only once a collator node has successfully [**linked a session key to its address**](./03_session_keys.md) and [**synced the parachain and relaychain states**](./05_sync_blockchain.md) by following the previous steps.
+:::
+
+## Economical Requirements
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/06_join_collators.md]: C: We could rename this section to something like "Minimum Token Requirement" or something like that?

File: docs/participate/01_Staking/01_Become a Collator/06_join_collators.md [hunk 26]
```diff
@@ -0,0 +1,58 @@
...
+
+The collator must call an extrinsic `parachainStaking -> joinCandidates(stake)` with the desired stake to join the candidate pool.
+
+<StakingTxDisclaimer />
+
+![](/img/chain/parachainStaking-joinCandidates.png)
+
+1. Select the collator's KILT address as the extrinsic submitter (the *using the selected account* field)
+2. Select the following extrinsic: `parachainStaking -> joinCandidates(stake)`
+3. Insert the staked KILT amount for the collator (any value between `10000000000000000000` and `200000000000000000000`)
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/06_join_collators.md]: Maybe having commas every third digit would help readability of this very long number.

File: docs/participate/01_Staking/01_Become a Collator/06_join_collators.md [hunk 27]
```diff
@@ -0,0 +1,58 @@
...
+4. Sign and submit the extrinsic (the *Submit Transaction* button)
+
+:::info
+A recent change in the blockchain metadata resulted in a change in the UI regarding how balances are shown.
+In the current version of PolkadotJS Apps, specifying 1 KILT requires adding 15 trailing `0`s.
+So, for instance, 1 KILT needs to be written as `1000000000000000`, while 10,000 KILT would be written as `10000000000000000000`.
+:::
+
+## How to check your position
```
Comment (ntn-x2) [inline docs/participate/01_Staking/01_Become a Collator/06_join_collators.md]: The `suggest` feature seems to be OFF for me right now. Could we add a `in the collators queue` or something like that? The way it is now seems a bit too generic for me.

File: docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md [hunk 28]
```diff
@@ -0,0 +1,63 @@
+---
+id: exit
+title: How to exit
```
Comment (ntn-x2) [inline docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md]: Also here, I would rename it to `Leave the set of collators` or `Leave the collator set`.

File: docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md [hunk 29]
```diff
@@ -0,0 +1,63 @@
...
+---
+
+import StakingTxDisclaimer from '../_disclaimer_staking_tx.md';
+
+When you intent to stop collating or just being a collator candidate, you have to go through three stages until your staked tokens are unlocked and your collator state is purged from the chain.
+
+:::info
+Unfortunately, exiting is not a simple process for security reasons.
+Since a picture says more than a thousand words, you can find a visualization of this process in the following [**lifecycle section**](./03_collator_lifecycle.md)
```
Comment (ntn-x2) [inline docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md]:
```suggestion
Since a picture says more than a thousand words, you can find a visualization of this process in the following [**lifecycle section**](./03_collator_lifecycle.md).
```

File: docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md [hunk 30]
```diff
@@ -0,0 +1,63 @@
...
+![](/img/chain/parachainStaking-cancelLeaveCandidates.png)
+
+1. Select the collators's KILT address as the extrinsic submitter (the *using the selected account* field)
+2. Select the appropriate extrinsic: `parachainStaking -> cancelLeaveCandidates`
+3. Sign and submit the extrinsic (the *Submit Transaction* button)
+
+## Unlock your stake
+
+In case you executed the exit request, you cannot immediately unlock your previously staked tokens.
+There is a delay of 7 days in block time before you can free them by calling `unlockUnstaked`.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md:62]:
```suggestion
There is a delay of ~7 days in block time before you can free them by calling `unlockUnstaked`.
```

File: docs/participate/01_Staking/02_Advanced Collator Section/06_benchmarking.md [hunk 31]
```diff
@@ -0,0 +1,40 @@
...
+:::
+
+The benchmarks can be run to compare the server's hardware capabilities against the referenced hardware.
+At the moment, we have benchmarked the Spiritnet and Peregrine runtimes on an AMD Ryzen 7 1700X with 64GB RAM and an NVMe SSD.
+After executing the benchmarks on a server compare the weights to the official KILT weights.
+Your weight results should at least be similar to the official ones and the lower yours are, the better.
+
+The commands executed to benchmark the KILT runtimes can be found in the official benchmark files for both [spiritnet](https://github.com/KILTprotocol/kilt-node/tree/master/runtimes/spiritnet/src/weights) and [peregrine](https://github.com/KILTprotocol/kilt-node/tree/master/runtimes/peregrine/src/weights).
+
+Below is an example of benchmarking for the the `balances` pallet.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/02_Advanced Collator Section/06_benchmarking.md:23]: M: The command refers to the staking pallet.

File: docs/participate/01_Staking/03_Delegate/01_overview.md [hunk 32]
```diff
@@ -0,0 +1,17 @@
+---
+id: overview
+title: Overview
+---
+
+In the [KILT **L**imited **D**elegated **P**roof **o**f **S**take (LDPoS)](https://medium.com/kilt-protocol/the-continuing-evolution-of-kilt-protocol-limited-delegated-proof-of-stake-640403427c48) consensus, delegators play an important role (at least in the infancy of the network) to filter the pool of candidates for honest, trusted and well-performing collators.
+
+The requirements to become a delegator are much more lightweight than [those for collators](../01_Become%20a%20Collator/01_overview.md). 
+You only need to stake a low number of tokens and decide on a collator candidate.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/03_Delegate/01_overview.md]:
```suggestion
You only need to stake a relatively lower number of tokens and decide on a collator candidate.
```

File: docs/participate/01_Staking/03_Delegate/02_become.md [hunk 33]
```diff
@@ -0,0 +1,99 @@
...
+
+:::info
+A recent change in the blockchain metadata resulted in a change in the UI regarding how balances are shown.
+In the current version of PolkadotJS Apps, specifying 1 KILT requires adding 15 trailing `0`s.
+So, for instance, 1 KILT needs to be written as `1000000000000000`, while 10,000 KILT would be written as `10000000000000000000`.
+:::
+
+
+## Happy path
+If your chosen collator candidate has at least one empty slot in their delegation pool (35 at the time of writing), your delegation is successful and you immediately receive rewards for each authored block of your collator.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/03_Delegate/02_become.md]:
```suggestion
If your chosen collator candidate has at least one empty slot in their delegation pool (out of 35 maximum slots at the time of writing), your delegation is successful and you immediately start receiving rewards each time the collator you delegated will produce a block.
```

File: docs/participate/01_Staking/03_Delegate/02_become.md [hunk 34]
```diff
@@ -0,0 +1,99 @@
...
+
+<div className="kilt-mermaid">
+
+```mermaid
+flowchart TD
+   A["Hold at least 20 KILT"] --> |"decide on candidate"| B("Collator Candidate chosen");
+   B --> |"call extrinsic joinDelegators"| C{"Can delegate to target? \n Either \n 1. There are empty \n delegations or \n 2. You delegate more \n than another delegator"};
```
Comment (ntn-x2) [inline docs/participate/01_Staking/03_Delegate/02_become.md:39]: Why are some sentences lowercase and some uppercase?

File: docs/participate/01_Staking/03_Delegate/04_revoke.md [hunk 35]
```diff
@@ -0,0 +1,29 @@
+---
+id: revoke
+title: How to revoke
```
Comment (ntn-x2) [inline docs/participate/01_Staking/03_Delegate/04_revoke.md]:
```suggestion
title: Revoke your delegation
```

File: docs/participate/01_Staking/03_Delegate/04_revoke.md [hunk 36]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+import StakingTxDisclaimer from '../_disclaimer_staking_tx.md';
+
+You can revoke your delegation at any time by calling `parachainStaking -> revokeDelegation`.
+As a result, you won't receive any rewards immediately after the transaction is successful.
```
Comment (ntn-x2) [inline docs/participate/01_Staking/03_Delegate/04_revoke.md]:
```suggestion
As a result, you won't receive any rewards immediately after the transaction is successfully executed.
```

File: docs/participate/01_Staking/03_Delegate/05_exit.md [hunk 37]
```diff
@@ -0,0 +1,24 @@
+---
+id: exit
+title: How to exit
```
Comment (ntn-x2) [inline docs/participate/01_Staking/03_Delegate/05_exit.md]:
```suggestion
title: Leave the set of delegators
```

File: docs/participate/01_Staking/04_unlock_unstaked.md [hunk 38]
```diff
@@ -0,0 +1,48 @@
+---
+id: unlock-unstaked
+title: How to unlock unstaked
```
Comment (ntn-x2) [inline docs/participate/01_Staking/04_unlock_unstaked.md]:
```suggestion
title: Unlock unstaked tokens
```

**Review by ntn-x2 — COMMENTED** [review]
This is a much better section than it was before!
I left quite some comments (sorry about that), but they are mostly nitpicks. The most important stuff is:

1. Whether we want to change all the `How to` titles to something that is more descriptive. Right now it is a bit of a mixed approach, but it could also be resolved in a separate editorial PR.
2. Probably switching the node setup instructions with the syncing/keys stuff
3. Probably merging few sections here and there, e.g., the sync and benchmarking sections.

File: docs/develop/01_sdk/02_Cookbook/04_Issue_and_verify_a_KILT_credential/01_ctypes.md (hunk 3 above)
Comment (wischli) [reply docs/develop/01_sdk/02_Cookbook/04_Issue_and_verify_a_KILT_credential/01_ctypes.md]: I merged incorrectly. But for some reason, the Github Frontend did not show me these files when I requested a review. Sorry.

File: docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md (hunk 30 above)
Comment (wischli) [reply docs/participate/01_Staking/02_Advanced Collator Section/02_exit.md:62]: It is exactly 7 days in block time, so there is no need to make this definition more vague 😉

File: docs/participate/01_Staking/03_Delegate/02_become.md (hunk 34 above)
Comment (wischli) [reply docs/participate/01_Staking/03_Delegate/02_become.md:39]: I decided to use lowercase for all actions (e.g. edges of the graph) and uppercase for the nodes. Do you think we should rather stay consistent?

Comment (ntn-x2) [reply docs/participate/01_Staking/03_Delegate/02_become.md:39]: No, if it was a deliberate decision than it's fine 😁

File: docs/participate/01_Staking/01_Become a Collator/04_setup_node.md (hunk 21 above)
Comment (wischli) [reply docs/participate/01_Staking/01_Become a Collator/04_setup_node.md:2]: I also thought about this. However, the node command already requires you to configure the path of the session keys. Thus, I think we should keep the session keys section before the node one.

File: docs/develop/02_chain/04_fullnode.md (hunk 8 above)
Comment (wischli) [reply docs/develop/02_chain/04_fullnode.md:91]: According to the [Polkadot Wiki](https://wiki.polkadot.network/docs/maintain-sync#running-an-archive-node), uppercase. Good catch!

Comment (ntn-x2) [reply docs/develop/02_chain/04_fullnode.md:91]: I mean also throughout the text, there are few occurrences with lower case and few with upper case.

File: docs/participate/01_Staking/01_Become a Collator/04_setup_node.md (hunk 21 above)
Comment (ntn-x2) [reply docs/participate/01_Staking/01_Become a Collator/04_setup_node.md:2]: Alright 👍

File: docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md (hunk 14 above)
Comment (wischli) [reply docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md]: Not as far as I know.

Comment (ntn-x2) [reply docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md]: I meant the six months of life. If we haven't passed the threshold yet then we could change six to twelve 😄

Comment (wischli) [reply docs/participate/01_Staking/01_Become a Collator/02_hardware_requirements.md]: A [recent backup of a Kusama Full Node is 129GB](https://paranodes.io/DBSize), so I changed the recommendation to be at least 250GB for the first year.

**Review by ntn-x2 — APPROVED** [review]
Let's go! 🔥
